ee/tabby-db/migrations/0029_merged-provider-tables.up.sql

```diff
@@ -3,6 +3,7 @@ CREATE TABLE integrations(
     kind TEXT NOT NULL,
     display_name TEXT NOT NULL,
     access_token TEXT NOT NULL,
+    url_base TEXT NOT NULL,
```

Comment:
```suggestion
    api_base TEXT,
```

Making this a nullable field, and for github / gitlab SAAS, it shall be null.

Please create separate `kind` enum for github self hosted / gitlab self hosted.